### ISSUE TITLE
Some way to identify/highlight duplicate image files?

### ISSUE THREAD

**Issue by kandakoj** [SpeciesFileGroup/taxonworks#2909]
### Feature or enhancement

Currently, I can see if an image is a duplicate file or not as a soft validation in the show image task, only for a specific image. 
As a user managing data from a number of contributors, it would be useful to have some task that highlights all sets of duplicate images in a project. Maybe as a quick fix, since the soft validation exists, can it be added as an option in the filter image task?

### Location

Filter Image Task

### Your role

researcher in small taxonomy-focused lab group

**Comment by mjy** [comment]
@kandakoj technically there can never be a duplicate image in the system based on its MD5, so I'm unsure as to the use case here.  Have you found duplicates in existing data?

**Comment by kandakoj** [comment]
I'm not sure what causes it, but I've been adding photos directly to collection objects in comprehensive.  When I use the filter image task, I'll see two copies of the image, one tied to the collecting object (as expected), and one that isn't associated with any collection object, and both copies have the soft validation about being a duplicate. 

Not sure what triggers it, sometimes I do delete a file and reupload it (or a different version) with the same name so maybe some indexing isn't completely cleared even if the file is deleted?

**Comment by mjy** [comment]
Indexing is attached to the Image, not the depiction, so if the image is gone the index is too.

Can you:
1) screenshot to show us what you mean by duplicate
2) Find two identical images by their TW ID and post them here?(like `/images/123` and `/images/456`)?

Indexing has nothing to do with filename, that doesn't matter, it has to do with the actual bits of the image.

**Comment by kandakoj** [comment]
So I dragged and dropped an identically named file in comprehensive, deleted the new version. It still appears in the filter image results. Click show. Right is the "originally" uploaded image (810228). Left is the file that I thought I deleted (810281). Depiction is no more, but image is still in the project .
![ScreenShot_duplicate](https://user-images.githubusercontent.com/70152690/163607186-ae065856-bee0-470c-aa56-0fbf87e06eb9.jpg)
